packages/server/api/routes/orderItems.router.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const orderItemsController = require('../controllers/orderItems.controller');
+
+router.post('/', (req, res) => {
+  orderItemsController
+    .addMovieToCart(req.body)
+    .then((result) => {
+      res.status(201).json(result);
+    })
+    .catch((error) => {
+      res.status(400).send('Bad request').end();
+      res.status(500).json({ error: error.message });
+    });
+});
+
+router.delete('/:userID/:movieID/:orderID', (req, res) => {
+  orderItemsController
+    .removeMovieFromCart(
+      req.params.userID,
+      req.params.movieID,
+      req.params.orderID,
+    )
+    .then((result) => {
```

Comment:
You could use async/await here instead. 
What you have written is correct but it's a slightly older syntax in javascript :)

Reply:
Thank you so much. 
I decided to keep .then method because I think its more readable for me